### PR TITLE
[ARCTIC-265][Site] Benchmark notebook

### PR DESCRIPTION
**Pull request by shidayang**
## Why are the changes needed?
Add docs of benchmark
Resolve: #265 
## How was this patch tested?
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] Run test locally before making a pull request

## Documentation

  - Does this pull request introduces a new feature? (yes / **no**)
  - If yes, how is the feature documented? (not applicable / **docs** / JavaDocs / not documented)

### CODE REVIEW

**Comment by zhoujinsong** [comment]
@shidayang Could you please rename the PR's title to match pattern `[ARCTIC-{issue number}][{Moduel}]{description}`

File: site/docs/ch/mysql-install.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
+### 物理机部署mysql
+
+
+
+>  mysql链接
+
+```
+ http://nos-yq.126.net/innosql-release/mysql-5.7.20-v3e-linux-x86_64.tar.gz
```
Comment (zhoujinsong) [inline site/docs/ch/mysql-install.md:8]: Do all user can access this url?
Does a open source MySQL version is more suitable for open source users?

Comment (shidayang) [reply site/docs/ch/mysql-install.md:8]: This url is public

File: site/README.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ This directory contains the source for the Arctic site.
 * Pages are maintained in markdown in the `docs/` folder
 * Links use bare page names: `[link text](target-page)`
 
+### Style
+* Proper nouns should start with a capital letter
```
Comment (zhoujinsong) [inline site/README.md:10]:
```suggestion
* Proper nouns should start with a capital letter, like Hadoop、Hive、Iceberg、Arctic
```

File: site/docs/ch/benchmark/benchmark-step.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+### Presto
+Hudi 的 rt 表的查询也就是实时查询目前不支持 Trino，只支持 Presto,如果需要测试 Hudi 需要安装配置 Presto。
+
+[Presto-Install](https://prestodb.io/docs/current/installation/deployment.html)
+
+配置 Hudi 插件：
+
+[Hudi-Plugin-Install](https://prestodb.io/docs/current/connector/hudi.html)
+
+### Ams
```
Comment (YesOrNo828) [inline site/docs/ch/benchmark/benchmark-step.md:32]: Uppercase: AMS?

File: site/docs/ch/benchmark/benchmark-step.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+Arctic 的元数据服务，当前使用0.4版本，如果需要测试 Arctic 需要安装此服务
+
+[Ams-Install](../meta-service/dashboard.md)
```
Comment (YesOrNo828) [inline site/docs/ch/benchmark/benchmark-step.md:35]: Uppercase: AMS?

File: site/docs/ch/benchmark/benchmark-step.md (hunk 3 above)
Comment (shidayang) [reply site/docs/ch/benchmark/benchmark-step.md:32]: 1

File: site/docs/ch/benchmark/benchmark-step.md (hunk 4 above)
Comment (shidayang) [reply site/docs/ch/benchmark/benchmark-step.md:35]: 1

**Comment by zhoujinsong** [comment]
@shidayang You may want to replace `Link #265 ` to `Resolve #265` to help github link this PR to issue.

**Review by zhoujinsong — APPROVED** [review]
LGTM.

**Pull request merged**